src/authV2/azext_authV2/_help.py

```diff
@@ -0,0 +1,113 @@
+  - name: Enable authentication on the app
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --enabled true
+  - name: Pin the runtime version of the app to 1.4.7
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --runtime-version 1.4.7
+  - name: Configure the app with file based authentication by setting the config file path
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --config-file-path auth.json
+  - name: Configure the app to redirect to AzureActiveDirectory's login page for unauthenticated requests
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --unauthenticated-client-action RedirectToLoginPage --redirect-provider AzureActiveDirectory
+  - name: Configure the app to listen to the forward headers X-FORWARDED-HOST and X-FORWARDED-PROTO
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --proxy-convention Standard
+"""
+
+helps['webapp auth set'] = """
+type: command
+short-summary: Sets the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Set the json saved in file auth.json as the auth settings for the web app.
+    text: >
+        az webapp auth set -g myResourceGroup -n myUniqueApp --body @auth.json
+"""
+
+helps['webapp auth config-version show'] = """
+type: command
+short-summary: Show the configuration version of the authentication settings for the webapp.
+examples:
+  - name: Show the configuration version of the authentication settings for the webapp (autogenerated)
+    text: >
+        az webapp auth config-version show --name MyWebApp --resource-group MyResourceGroup
+"""
+
+helps['webapp auth config-version revert'] = """
+type: command
+short-summary: Reverts the configuration version of the authentication settings for the webapp from v2 to v1 (classic).
+examples:
+  - name: Revert the configuration version of the authentication settings for the webapp from v2 to v1 (classic) (autogenerated)
+    text: >
+        az webapp auth config-version revert --name MyWebApp --resource-group MyResourceGroup
+"""
+
+helps['webapp auth config-version upgrade'] = """
+type: command
+short-summary: Upgrades the configuration version of the authentication settings for the webapp from v1 (classic) to v2.
+examples:
+  - name: Upgrades the configuration version of the authentication settings for the webapp from v1 (classic) to v2 (autogenerated)
+    text: >
+        az webapp auth config-version upgrade --name MyWebApp --resource-group MyResourceGroup
+"""
+
+helps['webapp auth-classic'] = """
+type: group
+short-summary: Manage webapp authentication and authorization in the classic format.
+"""
+
+helps['webapp auth-classic show'] = """
+type: command
+short-summary: Show the authentification settings for the webapp in the classic format.
```

Comment:
What's the use-case for this command? Do we expect v2 users to use this command? If not, I think we should make the description more clear, but I'll wait to hear back before making a suggestion.

Reply:
if they have the extension installed and still want to interact with v1 apps they don't intend to upgrade this would be used. I guess it depends on whether we still need to offer backwards compatibility with the v1 experience.